docs/data-sources/connector_oci_vcn.md

```diff
@@ -0,0 +1,31 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "alkira_connector_oci_vcn Data Source - terraform-provider-alkira"
+subcategory: ""
+description: |-
+  Use this data source to get an existing OCI VCN connector.
+---
+
+# alkira_connector_oci_vcn (Data Source)
+
+Use this data source to get an existing OCI VCN connector.
+
+```terraform
```

Comment:
The example you need to add it to `example` directory with the data source name :) Otherwise, it will generate a diff when you do `make doc`.